vue.config.js

```diff
@@ -25,6 +25,12 @@ module.exports = {
     if (process.env.NODE_ENV === 'test') {
       config.resolve.alias.set('vue$', 'vue/dist/vue.esm-bundler.js');
     }
+
+    config.plugin('define').tap(definitions => {
+      // eslint-disable-next-line no-param-reassign
+      definitions[0].__VUE_PROD_HYDRATION_MISMATCH_DETAILS__ = JSON.stringify(false);
```

Comment:
I think you mean to revert this, but I thought I'd add a comment. I actually sort of like that it's telling us about issues like getodk/central#1029. But I also think that once we move to Vitest, this warning will probably go away entirely.